Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  OnboardingViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewControllerDidTapNextButton(viewController: OnboardingViewController)
```

Comment:
I suggest you rename to `onboardingViewControllerDidTapNext(_ viewController: OnboardingViewController)` to more closely follow Swift conventions.